internal/rsl/rsl.go

```diff
@@ -27,6 +27,9 @@ const (
 	EntryIDKey                 = "entryID"
 	SkipKey                    = "skip"
 
+	// PolicyStagingRef defines the Git namespace used as a staging area when creating or updating gittuf policies.
+	PolicyStagingRef = "refs/gittuf/policy-staging"
```

Comment:
why is this necessary in the rsl package?

Reply:
Removed